packages/api/src/external/carequality/command/cq-directory/cq-gateways.ts

```diff
@@ -0,0 +1,36 @@
+import { Op } from "sequelize";
+import { CQDirectoryEntryModel } from "../../models/cq-directory";
+
+export async function updateCQGateways(gateways: string[]): Promise<void> {
+  console.log(`Found ${gateways.length} gateways in the CQ directory. Updating...`);
+  await Promise.allSettled(gateways.map(gateway => updateCQGateway(gateway)));
+}
+
+export async function updateCQGateway(gateway: string): Promise<void> {
+  await CQDirectoryEntryModel.update(
+    {
+      gateway: true,
+    },
+    {
+      where: {
+        name: {
+          [Op.like]: `%${gateway}%`,
+        },
+        urlXCPD: {
+          [Op.ne]: "",
+        },
```

Comment:
Yea, I guess I had to use an empty string because of the other code you pointed out where I was assigning them [empty strings](https://github.com/metriport/metriport/pull/1575#discussion_r1486333939). Will fix this. 

The logic for not updating them is that gateways will be organizations like athena, epic, etc - large structures that we can send PD requests to. If an organization doesn't have `urlXCPD`, it is not a gateway